components/layout_2020/flexbox/layout.rs

```diff
@@ -787,9 +787,15 @@ impl FlexContainer {
             let align_content_style = flex_context.config.align_content.0.primary();
 
             // Inital values from the style system
```

Comment:
Nit: This comment was already misleading ("initial" as before resolving vs https://drafts.csswg.org/css-cascade/#initial-values), but it even makes less sense now.

Amend as in my suggested edit or just remove it.